packages/create-cloudflare/src/help.ts

```diff
@@ -31,8 +31,16 @@ export const showHelp = ({
 
 	logRaw(bold("OPTIONS\n"));
 
+	if (args?.experimental) {
+		logRaw(
+			blue(
+				"You have selected experimental mode - the options below are filtered to those that support experimental mode.\n",
```

Comment:
@tanushree-sharma - you might have a view on the copy here, but I wanted to call out to people that they would see a different list of frameworks/templates when the are running `--experimental` mode.